activesupport/test/core_ext/object/try_test.rb

```diff
@@ -96,4 +96,14 @@ def private_method
 
     assert_nil klass.new.try(:private_method)
   end
+
+  def test_try_with_delegated_method
+    klass = Class.new(SimpleDelegator) do
+      def delegated_method
+        'delegated method'
+      end
+    end
+
+    assert_equal 'delegated method', klass.new(Object.new).try(:delegated_method)
+  end
```

Comment:
I think we probably need more test cases here. At minimum, it should make sure the "falling through to the delegated object" case works